core/tests/ui/e2e/wishlist.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { expect, test } from '~/tests/fixtures';
+
+test('Guest user is redirected to login upon adding product to wishlist', async ({ page }) => {
+  await page.goto('/laundry-detergent/');
+  await page.getByRole('heading', { level: 1, name: '[Sample] Laundry Detergent' }).waitFor();
+
+  await page.getByRole('link', { name: 'Save to wishlist' }).click();
+
+  await expect(page).toHaveURL(/login/);
+});
+
+test('Favorites wishlist present by default and cannot be deleted', async ({ page, account }) => {
+  const customer = await account.create();
+
+  await customer.login();
+
+  await page.goto('/account/wishlists/');
+
+  await expect(page.getByText('Favorites')).toBeVisible();
+  await expect(page.getByRole('button', { name: 'Delete' })).toBeHidden();
+
+  await page.getByRole('link', { name: 'Favorites' }).click();
```

Comment:
L21-22 feels like it's incomplete. I would rather remove that for now and just leave a comment to suggest(like you added below) that we need to add missing part of the test.

Reply:
Makes sense, done!